core/services/fluxmonitorv2/key_store.go

```diff
@@ -0,0 +1,36 @@
+package fluxmonitorv2
+
+import (
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/common"
+	corestore "github.com/smartcontractkit/chainlink/core/store"
+)
+
+//go:generate mockery --name KeyStoreInterface --output ./mocks/ --case=underscore
+
+// KeyStoreInterface defines an interface to interact with the keystore
+type KeyStoreInterface interface {
+	Accounts() []accounts.Account
+	GetRoundRobinAddress() (common.Address, error)
+}
+
+// KeyStore implements KeyStoreInterface
+type KeyStore struct {
```

Comment:
We would still need access the store to get call the `GetRoundRobinAddress` function because `Accounts` is on `store.Keystore` but `GetRoundRobinAddress` in on the `store` itself. In order to make the change I would need to create 2 interfaces `KeyStore` which would have 

```
type KeyStore interface {
       Accounts() []accounts.Account 
}
```

and a separate interface for the `GetRoundRobinAddress` function (which I can't really think of a name for)

```
type AddressFetcher interface {
       GetRoundRobinAddress() (common.Address, error) 
}
```

Thoughts?